.github/workflows/code-style.yml

```diff
@@ -0,0 +1,19 @@
+name: code-style
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  VerifyFormatting:
+    runs-on: ubuntu-20.04
+
+    steps:
```

Comment:
Unless you want to have a separate status report and shield/badge for showing the status of this specific test/job independently from others, I would suggest not to create this new workflow. Instead, just copy this job (L7-L19) into the existing workflow.

Reply:
Done. I mostly started the new file while experimenting with the new flow, but merging it now makes it simpler.